views/partials/table.html

```diff
@@ -0,0 +1,18 @@
+<table>
+  <head>
+    <tr>
+      {{#headers}}
+        <th>{{.}}</th>
+      {{/headers}}
+    </tr>
+  </head>
+  <tbody>
+    {{#rows}}
+      <tr>
+        {{#.}}
+          <td>{{.}}</td>
+        {{/.}}
```

Comment:
@JoeChapman related to your earlier comment - I am kinda surprised this works :) - rows is [[cells], [cells]]